pkg/build/buildutil/util.go

```diff
@@ -36,9 +36,13 @@ const (
 	BuildBlobsContentCache = "/var/cache/blobs"
 
 	// buildPodSuffix is the suffix used to append to a build pod name given a build name
-	buildPodSuffix           = "build"
-	caConfigMapSuffix        = "ca"
-	sysConfigConfigMapSuffix = "sys-config"
+	buildPodSuffix          = "build"
+	caConfigMapSuffix       = "ca"
+	globalCAConfigMapSuffix = "global-ca"
+	// GlobalCAConfigMapAnnotation is the annotation key to set on a config map so that the platform's
+	// proxy support will inject any required CA's needed for proxy communication.
+	GlobalCAConfigMapAnnotation = "config.openshift.io/inject-proxy-cabundle"
```

Comment:
I did not change `inject-proxy-cabundle` because I did not see that the annotation name had changed yet in the design doc

Reply:
Makes sense - I'd expect there will be more churn here, but keep it for now. Design is _evolving_ to say the least.